2-mandatory/3-groceries.js

```diff
@@ -26,14 +26,15 @@ Exercise 1:
   The weeklyGroceriesToBuy array shouldn't contain any repeating items.
 */
 // Gather all week item names into this array
-let weeklyGroceriesToBuy = [];
+let weeklyGroceriesToBuy = [...new Set(Object.values(weeklyMealPlan).flat(1))];
```

Comment:
Nice way to get the get the unique values for an array by using spread operation along with new Set(). 
I think , there is no need to use flat method.